internal/cmd/commands/daemon/server.go

```diff
@@ -191,8 +184,16 @@ func (s *cacheServer) serve(ctx context.Context, cmd commander, l net.Listener)
 	}
 	mux.HandleFunc("/v1/personas", personaFn)
 
+	logger, err := event.SysEventer().StandardLogger(ctx, "daemon.serve: ", event.ErrorType)
```

Comment:
what if the eventer is nil?

Reply:
Isn't setupLogging(...) called in Run(...)?  I'd be happy if it was called in newServer(...).

Reply:
ty!